src/components/PuzzleBoard/puzzleBoard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class PuzzleBoard extends React.Component {
+  static defaultProps = {
+    puzzle: [],
+    correctlyGuessedLetters: [],
+    specialChars: []
+  };
+
+  renderPuzzle = () => {
+    const { puzzle, correctlyGuessedLetters, specialChars } = this.props;
+    return puzzle.map((letter, index) => {
+      if (
+        // correctly guessed letters
+        correctlyGuessedLetters.indexOf(letter) !== -1 ||
+        specialChars.indexOf(letter) !== -1
+      ) {
+        return <span key={index}>{letter}</span>;
+      } else if (letter === " ") {
+        return <span key={index}>&nbsp;&nbsp;&nbsp;</span>;
+      } else {
+        // unguessed letters
+        return <span key={index}>__ </span>;
+      }
+    });
+  };
+
+  render() {
+    return <div className="puzzleBoard">{this.renderPuzzle()}</div>;
```

Comment:
This might be an item to address in a future PR, but it would be nice if this component owned all the styles that are connected to  `className="puzzleBoard"`. Perhaps a new css file called `puzzleBoard.css` in this same directory.

Right now, this component is sharing responsibility for styling with App.js and the main stylesheet.

Put another way, `<PuzzleBoard />`, and no one else, should ideally own and be responsible for everything within its domain, which includes styling.